ci/test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+
+## libelf1 is needed to run flow
```

Comment:
oh, we still use Flow here? 😅

Reply:
apparently yes, I didn't change anything 😛